content/200-orm/300-prisma-migrate/300-workflows/20-prototyping-your-schema.mdx

```diff
@@ -148,30 +148,12 @@ The following scenario demonstrates how to use `db push` to synchronize a new sc
     npx prisma db push
     ```
 
-    `db push` will prompt you to reset because you cannot add a required field to a table with existing content unless you provide a default value:
-
-    ```bash
-    ⚠️ We found changes that cannot be executed:
-
-    • Added the required column `description` to the `Post` table without a default value. There are 2 rows in this table, it is not possible to execute this.
-
-    ? To apply this step we need to reset the database, do you want to continue? All data will be lost. » (y/N)
-    ```
-
-:::tip
-
-Use the `--accept-data-loss` flag to skip this warning, or `--force-reset` to ignore all warnings.
-
-:::
+    `db push` will fail because you cannot add a required field to a table with existing content unless you provide a default value.
 
 6.  Confirm data loss and apply changes to your database (or revisit your schema):
 
-    ```bash
-     There might be data loss when applying the changes:
-
-     • Added the required column `description` to the `Post` table without a default value.
-
-     ? Do you want to ignore the warning(s)? Some data will be lost. » (y/N)
+    ```terminal
+    npx prisma migrate reset
```

Comment:
The previous line says "confirm data loss" - that's not really accurate any more right?